content/en/docs/tasks/cluster-administration/node-overprovisioning/node-overprovisioning.md

```diff
@@ -23,9 +23,7 @@ By maintaining some unused capacity, you ensure that resources are immediately a
 
 Begin by creating a Deployment that runs placeholder pods. These pods should use a minimal process, such as a pause container, and have a very low priority to ensure they can be easily preempted when higher-priority pods require resources.
 
-To define a Low-Priority Priority Class
-
-Create a Priority Class with a low value to assign to the placeholder pods.
+To define a Low-Priority Priority Class. First, create a Priority Class with a low value to assign to the placeholder pods.
```

Comment:
Don't spell "Priority Class", we either talk about "priority class" as a concept or "PriorityClass" as a resource kind.